tests/st/runtime/test_dynamic_shape.py

```
@@ -265,25 +264,25 @@ def compute_expected(self, tensors, params=None):
 class TestDynamicShapeOperations:
     """Test suite for dynamic shape kernel operations."""
 
-    @pytest.mark.parametrize("backend", PLATFORMS)
+    @pytest.mark.parametrize("platform", PLATFORMS)
     @pytest.mark.parametrize("shape", _SHAPES)
-    def test_dyn_shape_add(self, test_runner, shape, backend):
+    def test_dyn_shape_add(self, test_runner, shape, platform):
         """Test add with fully dynamic M×N tensor shapes."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)